changelog.d/5915.bugfix

```diff
@@ -0,0 +1 @@
+The right filesystempath is now used to store thumbnails. Fix reported by rkfg.
```

Comment:
please could you phrase this in terms of the change in behaviour that the user will see?

(as per my questions on #5913, I remain confused as to what bug we are trying to fix here)

Reply:
have changed it. Can you take a look if that is more fitting?

Reply:
sounds about right to me. thanks!